website/docs/Pipelines/ImputationBeagle_Pipeline/_category_.json

```diff
@@ -0,0 +1,4 @@
+{
+  "label": "ImputationBeagle",
```

Comment:
im not sure if this is correct and im not sure what the value for `position` should be